### PR TITLE
added get load and clear code

### PR DESCRIPTION
**Pull request by gcezar**

### CODE REVIEW

File: chargepoint/chargepoint_api.py [hunk 1]
```diff
@@ -4,50 +4,41 @@
...
-#         # print('Row Session: ', row_session)
-#     except:
-#         pass
-
 class CP_API:
     def __init__(self):
         self.username = '92d0023b4f07baa1857381a51de78772546674ef2b8391416000751'
         self.password = 'c2f058fc77298198f5bb3089cb354053'
         self.wsdl_url = "https://webservices.chargepoint.com/cp_api_5.0.wsdl"
         self.client = Client(self.wsdl_url, wsse=UsernameToken(self.username, self.password))
```
Comment (jongoncalves) [inline chargepoint/chargepoint_api.py:13]: where does `Client` come from?

File: chargepoint/chargepoint_api.py [hunk 2]
```diff
@@ -4,50 +4,41 @@
...
 
-    def getSession(self, tStart, tEnd):
 
-        usageSearchQuery = {'fromTimeStamp': tStart, 'toTimeStamp': tEnd,}
-        data = self.client.service.getChargingSessionData(usageSearchQuery)
-        print('session data: ', data['ChargingSessionData'])
-        df = pd.DataFrame(data['ChargingSessionData'])
-        print(df.head())
+    def getChargingSessionData(self, tStart, tEnd, stationID='null'):
```
Comment (jongoncalves) [inline chargepoint/chargepoint_api.py]: naming in python -> `snake_case` and not `camelCase`

File: chargepoint/chargepoint_api.py [hunk 3]
```diff
@@ -4,50 +4,41 @@
...
+# SESSION
 tStart = datetime(2020, 2, 13, 00, 00, 00)
 tEnd = tStart + timedelta(hours=23, minutes=59, seconds=59)
-cp.getSession(tStart, tEnd)
\ No newline at end of file
+session = cp.getChargingSessionData(tStart, tEnd, stationID = '1:113189')
+print(session['sessionID'])
+
+# LOAD
+Load = []
```
Comment (jongoncalves) [inline chargepoint/chargepoint_api.py]: naming

File: chargepoint/chargepoint_api.py [hunk 4]
```diff
@@ -4,50 +4,41 @@
...
+stationID = ['1:113189', '1:112413', '1:113177']
+for i in stationID:
+    Load.append(cp.getLoad(i))
+print(Load[0])
```
Comment (jongoncalves) [inline chargepoint/chargepoint_api.py]: remove print statements

File: chargepoint/chargepoint_api.py (hunk 1 above)
Comment (gcezar) [reply chargepoint/chargepoint_api.py:13]: It comes from `from zeep import Client` in line 1

**Pull request merged**